spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -208,19 +241,36 @@ def call_with_params(params)
 
     it "renders new if the work has invalid pseuds" do
       work_attributes = attributes_for(:work).except(:posted)
-      work_attributes[:author_attributes] = { ids: @user.pseud_ids,
+      work_attributes[:author_attributes] = { ids: user.pseud_ids,
                                               byline: "*impossible*" }
       post :create, params: { work: work_attributes }
       expect(response).to render_template("new")
       expect(assigns[:work].errors.full_messages).to \
         include "Invalid creator: Could not find a pseud *impossible*."
     end
 
+    xit "renders new if edit is pressed" do
```

Comment:
Why is this test disabled? Is the test broken, or is there something wrong with the code it's checking?

Reply:
I think it was broken in an earlier version but is now working